swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
+from .item import Item
+class Clothing(Item):
+    def __init__(self, condition = None, category = "Clothing"):
+        super().__init__(category, condition)
```

Comment:
one thing we could do is hardcode "clothing" because as a Clothing child class, it's always supposed to be "clothing", right? So, let's not even give the user tha chance to mess that up!

```suggestion
    def __init__(self, condition = None):
        super().__init__("Clothing", condition)
```